appveyor_test.sh

```diff
@@ -2,6 +2,6 @@
 
 COLD_OCAML="$(sed -ne 's/URL_ocaml = .*ocaml-.*ocaml-\(.*\)\.tar\.gz/ocaml.\1/p' src_ext/Makefile)"
 # This is supposed to be able to select ocaml-system using the bootstrap compiler
-opam init -y -a --compiler=$COLD_OCAML git+$OPAM_REPO#$OPAM_TEST_REPO_SHA
+opam init -y -a --compiler=$COLD_OCAML git+https://github.com/ocaml/opam-repository#$OPAM_TEST_REPO_SHA
```

Comment:
This line was correct - we need this line from master in appveyor.yml:

https://github.com/ocaml/opam/blob/a2c0e226b9d47fd972dacd2d9387f2f3420a8412/appveyor.yml#L15